<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ucTabContaint</class>
 <widget class="QWidget" name="ucTabContaint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>644</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>685</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gbSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbFileOpen">
        <property name="text">
         <string>Open File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbFileOpen">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>285</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>23</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btFileOpen">
        <property name="minimumSize">
         <size>
          <width>86</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>86</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>File ...</string>
        </property>
        <property name="icon">
         <iconset resource="Images.qrc">
          <normaloff>:/Icons/48x48/Images/48x48/folder.png</normaloff>:/Icons/48x48/Images/48x48/folder.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>125</width>
          <height>22</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="btStart">
        <property name="minimumSize">
         <size>
          <width>86</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>86</width>
          <height>25</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Start !</string>
        </property>
        <property name="icon">
         <iconset resource="Images.qrc">
          <normaloff>:/Icons/48x48/Images/48x48/play.png</normaloff>:/Icons/48x48/Images/48x48/play.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbFilter">
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="tbFilter">
        <property name="minimumSize">
         <size>
          <width>285</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="ckEnableFilter">
        <property name="text">
         <string>Enable Filter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="btSetHighLight">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>107</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Highlight</string>
        </property>
        <property name="icon">
         <iconset resource="Images.qrc">
          <normaloff>:/Icons/128x128/Images/128x128/highlighter.png</normaloff>:/Icons/128x128/Images/128x128/highlighter.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="btOpenFile">
        <property name="minimumSize">
         <size>
          <width>86</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>86</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Open File</string>
        </property>
        <property name="icon">
         <iconset resource="Images.qrc">
          <normaloff>:/Icons/48x48/Images/48x48/open_file.png</normaloff>:/Icons/48x48/Images/48x48/open_file.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbFile">
     <property name="title">
      <string>File</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="teFile">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>244</red>
              <green>244</green>
              <blue>244</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Courier</family>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="lineWrapColumnOrWidth">
         <number>0</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbFileOpen</tabstop>
  <tabstop>btFileOpen</tabstop>
  <tabstop>btStart</tabstop>
  <tabstop>tbFilter</tabstop>
  <tabstop>ckEnableFilter</tabstop>
  <tabstop>btOpenFile</tabstop>
  <tabstop>btSetHighLight</tabstop>
  <tabstop>teFile</tabstop>
 </tabstops>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btStart</sender>
   <signal>clicked()</signal>
   <receiver>ucTabContaint</receiver>
   <slot>ActionButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btFileOpen</sender>
   <signal>clicked()</signal>
   <receiver>ucTabContaint</receiver>
   <slot>FileOpenClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSetHighLight</sender>
   <signal>clicked()</signal>
   <receiver>ucTabContaint</receiver>
   <slot>ShowSelectHighLight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btOpenFile</sender>
   <signal>clicked()</signal>
   <receiver>ucTabContaint</receiver>
   <slot>OpenInEditor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>664</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbFileOpen</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>ucTabContaint</receiver>
   <slot>SelectFileChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ckEnableFilter</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ucTabContaint</receiver>
   <slot>FilterCheckChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbFilter</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ucTabContaint</receiver>
   <slot>FilterCleared(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>-3</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ActionButtonClicked()</slot>
  <slot>FileOpenClicked()</slot>
  <slot>ShowFileClicked()</slot>
  <slot>ShowSelectHighLight()</slot>
  <slot>OpenInEditor()</slot>
  <slot>SelectFileChanged(QString)</slot>
  <slot>FilterCheckChange(int)</slot>
  <slot>FilterCleared(QString)</slot>
 </slots>
</ui>
